### PR TITLE
Changed the help text maya into Mayactl

### LINKED ISSUE #354: changing maya to mayactl in helptext of mayactl
## Is this a BUG REPORT or FEATURE REQUEST?
 
 Choose one: BUG REPORT or FEATURE REQUEST

**What happened**:

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Maya version (use `maya version`):
- M-apiserver version (use `m-apiserver version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by balakeshbalu**
Signed-off-by: balakeshbalu <balakeshjnvkp22@gmail.com>

**What this PR does / why we need it**:
This PR changes the term maya from mayctl in helptext of mayactl volume related files

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #354 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by balakeshbalu** [comment]
```
Usage: mayactl volume <subcommand> [options] [args]

This command provides operations related to a Volume.

Create a Volume:
$ mayactl volume create --volname <vol> -size <size>

List Volumes:
$ mayactl volume list

Delete a Volume:
$ mayactl volume delete --volname <vol>

Statistics of a Volume:
$ mayactl volume stats --volname <vol>


```

File: cmd/mayactl/app/command/snapshot/create.go [hunk 1]
```diff
@@ -46,6 +46,16 @@ import (
...
 // CmdSnaphotCreateOptions holds the options for snapshot
 // create command
+var (
+	snapshotCreateCommandHelpText = `
+	usage: mayactl snapshot create --volname <vol> --snapname <snap>
+	
+	this command creates a volume snapshot 
+	
+	note: the volume should exit before itself
+	`
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/snapshot/create.go]: nit : Remove the extra lines

File: cmd/mayactl/app/command/snapshot/revert.go [hunk 2]
```diff
@@ -30,13 +30,23 @@ import (
 }*/
 
 // NewCmdSnapshotRevert reverts a snapshot of OpenEBS Volume
+var (
+	snapshotRevertCommandHelpText = `
+	usage: mayactl snapshot revert --volname <vol> --snapname <snap>
+	
+	this command revert a volume snapshot 
+
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/snapshot/revert.go]: nit : Remove the extra lines

File: cmd/mayactl/app/command/snapshot/snapshot.go [hunk 3]
```diff
@@ -20,11 +20,29 @@ import (
...
+
+	Create a snapshot:
+	$ mayactl snapshot create --volname <vol> --snapname <snap>
+
+	List snapshot:
+	$ mayactl snapshot list  --volname <vol>
+
+	revert snapshot:
+	$ mayactl snapshot revert --volname <vol> --snapname <snap>
+
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/snapshot/snapshot.go]: nit : Remove the extra lines

File: cmd/mayactl/app/command/volume.go [hunk 4]
```diff
@@ -22,21 +22,21 @@ import (
...
-	Usage: maya volume <subcommand> [options] [args]
+Usage: mayactl volume <subcommand> [options] [args]
 
-	This command provides operations related to a Volume.
+This command provides operations related to a Volume.
 
-	Create a Volume:
-	$ maya volume create -volname <vol> -size <size>
+Create a Volume:
+$ mayactl volume create --volname <vol> -size <size>
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/volume.go]: `-size` --> `--size`

File: cmd/mayactl/app/command/volume_create.go [hunk 5]
```diff
@@ -27,15 +27,15 @@ import (
...
 
-	This command creates a new Volume.
+This command creates a new Volume.
 
-	Volume create options:
-	-size
-	Provisioning size of the volume(default is 5G)
+Volume create options:
+-size
+Provisioning size of the volume(default is 5G)
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/volume_create.go]: Not needed, flags info will be added by sub-method itself , so delete this part

File: cmd/mayactl/app/command/commands.go [hunk 6]
```diff
@@ -25,8 +25,8 @@ import (
 func NewMayaCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "mayactl",
-		Short: "Maya means 'Magic'a tool for storage orchestration",
-		Long:  `Maya means 'Magic' a tool for storage orchestration`,
+		Short: "Mayactl means 'Magic'a tool for storage orchestration",
```
Comment (mahebbar) [inline cmd/mayactl/app/command/commands.go]: 'Magic' <add space> a tool

File: cmd/mayactl/app/command/snapshot/create.go [hunk 7]
```diff
@@ -46,6 +46,15 @@ import (
 
 // CmdSnaphotCreateOptions holds the options for snapshot
 // create command
+var (
+	snapshotCreateCommandHelpText = `
+usage: mayactl snapshot create --volname <vol> --snapname <snap>
+
+this command creates a volume snapshot
```
Comment (mahebbar) [inline cmd/mayactl/app/command/snapshot/create.go]: The following command...

File: cmd/mayactl/app/command/volume_delete.go [hunk 8]
```diff
@@ -27,11 +27,11 @@ import (
 
 var (
 	volumeDeleteCommandHelpText = `
-	Usage: maya volume delete -volname <vol>
+Usage: mayactl volume delete --volname <vol>
 
-	This command initiate a delete for OpenEBS Volume.
+This command initiate a delete for OpenEBS Volume.
```
Comment (mahebbar) [inline cmd/mayactl/app/command/volume_delete.go]: This command initiates a deletion process for an OpenEBS Volume.

File: cmd/mayactl/app/command/volume_info.go [hunk 9]
```diff
@@ -16,11 +16,11 @@ import (
...
 var (
 	volumeInfoCommandHelpText = `
-	    Usage: mayactl volume info --volname <vol>
+Usage: mayactl volume info --volname <vol>
 
-        This command fetches the information and status of the various
-	    aspects of the Volume such as ISCSI, Controller and Replica.
-        `
+This command fetches the information and status of the various
+aspects of the Volume such as ISCSI, Controller and Replica.
```
Comment (mahebbar) [inline cmd/mayactl/app/command/volume_info.go]: of a Volume... Controller, and Replica.

File: cmd/mayactl/app/command/volume_info.go [hunk 10]
```diff
@@ -16,11 +16,11 @@ import (
 
 var (
 	volumeInfoCommandHelpText = `
-	    Usage: mayactl volume info --volname <vol>
+Usage: mayactl volume info --volname <vol>
 
-        This command fetches the information and status of the various
-	    aspects of the Volume such as ISCSI, Controller and Replica.
-        `
+This command fetches the information and status of the various
```
Comment (mahebbar) [inline cmd/mayactl/app/command/volume_info.go]: fetches information

File: cmd/mayactl/app/command/volume_stats.go [hunk 11]
```diff
@@ -35,15 +35,11 @@ import (
 
 var (
 	volumeStatsCommandHelpText = `
-	Usage: maya volume stats <vol> [-size <size>]
+Usage: mayactl volume stats --volname <vol> [-size <size>]
 
-	This command queries the stats of the volume.
+This command queries the stats of the volume.
```
Comment (mahebbar) [inline cmd/mayactl/app/command/volume_stats.go]: stats = status

File: cmd/mayactl/app/command/volumes_list.go [hunk 12]
```diff
@@ -27,10 +27,10 @@ import (
 
 var (
 	volumesListCommandHelpText = `
-	Usage: maya volume list [options]
+Usage: mayactl volume list [options]
 
-	This command displays status of available Volumes.
-	If no volume ID is given, a list of all known volume will be dumped.
+This command displays status of available Volumes.
+If no volume ID is given, a list of all known volume will be dumped.
```
Comment (mahebbar) [inline cmd/mayactl/app/command/volumes_list.go]: known volumes...

**Comment by prateekpandey14** [comment]
closing this wrt https://github.com/openebs/maya/pull/367

**Pull request closed without merging**